cedar-policy/src/api.rs

```diff
@@ -1169,21 +1383,38 @@ pub struct EntityTypeName(ast::Name);
 
 impl EntityTypeName {
     /// Get the basename of the `EntityTypeName` (ie, with namespaces stripped).
+    /// ```
+    /// use cedar_policy::EntityTypeName;
+    /// use std::str::FromStr;
```

Comment:
let's hide these imports
```suggestion
    /// # use cedar_policy::EntityTypeName;
    /// # use std::str::FromStr;
```